pipeline/tests/pil.rs

```diff
@@ -1,4 +1,3 @@
-#[cfg(feature = "halo2")]
```

Comment:
I think that was just wrong? With this, you can't run tests without the `halo2` feature.

Reply:
This only applies to the next item, not the whole mod.

Reply:
Yeah, exactly, but why would we import Goldilocks only if the `halo2` feature is enabled?